docs/source/en/using-diffusers/schedulers.md

```diff
@@ -241,7 +258,7 @@ image
 </p>
 
 
-At the time of writing this doc [`DPMSolverMultistepScheduler`] gives arguably the best speed/quality trade-off and can be run with as little 
+At the time of November 15th, 2022 [`DPMSolverMultistepScheduler`] gives arguably the best speed/quality trade-off and can be run with as little 
```

Comment:
Let's go with this to avoid dating the doc. cc @patrickvonplaten @sayakpaul @yiyixuxu is this statement still valid?

```suggestion
[`DPMSolverMultistepScheduler`] gives one of the best speed/quality trade-offs and can be run with as little 
```

Reply:
I would write "a reasonable trade-off".